src/app/index.scss

```diff
@@ -63,6 +63,7 @@ button:focus-visible {
 	outline: 4px auto -webkit-focus-ring-color;
 }
 
+
```

Comment:
Why is this change is here?

Reply:
Essentially, nothing has changed here compared to what is now in the master.
The empty line was added because I first removed all styles for button in this file, and then added them back (with this empty line).

Reply:
if something like that happens in the future, simply return the file into its initial state either with the corresponding git command or using the UI features of your IDE. 

those empty lines adds the number of files to review when there's nothing to review really, so it is both common sense and basic ethics to keep track of removing unnecessary changes from your PRs